.gitignore

```diff
@@ -31,7 +31,8 @@ yarn-error.*
 # macOS
 .DS_Store
 *.pem
-
+.yarn
```

Comment:
ive been using bun

Reply:
in our system we using yarn so i have added that in ignore file